modules/Microsoft.DBforPostgreSQL/flexibleServers/.test/private/dependencies.bicep

```diff
@@ -0,0 +1,69 @@
+@description('Optional. The location to deploy to.')
+param location string = resourceGroup().location
+
+@description('Required. The name of the Virtual Network to create.')
+param virtualNetworkName string
+
+@description('Required. The name of the Managed Identity to create.')
+param managedIdentityName string
+
+@description('Required. The name of the Private DNS Zone to create.')
```

Comment:
To align with other modules I'd suggest to remove this input parameter and hardcode it in the dependencies template